akka-actor/src/main/scala/akka/util/Reflect.scala

```diff
@@ -154,4 +154,25 @@ private[akka] object Reflect {
       }
     }
   }
+
+  /**
+   * INTERNAL API
+   */
+  private[akka] def findClassLoader(): ClassLoader = {
+    def findCaller(get: Int ⇒ Class[_]): ClassLoader =
+      Iterator.from(2 /*is the magic number, promise*/ ).map(get) dropWhile { c ⇒
```

Comment:
Ok, I dig a bit on it, the method recourse over the call stack to find the ClassLoader.
Method calls obviously matter but in this case calling objects that starts with ```akka.util.Reflect``` are dropped, so what happens with this implementation is that one more object get discarded during search.

Writing a test case here will be a bit wired, because everything should be executed within a Thread with the context class loader ```null```, probably this is a matter of Security environments but hard to reproduce and verify.

Is an accurate debug enough or I should write test case too?